.github/workflows/main.yml

```diff
@@ -128,15 +128,56 @@ jobs:
 
   publish-flextesa-plugin-to-npm:
     needs:
-      - publish-ligo-plugin-to-npm
-      - publish-smartpy-plugin-to-npm
-      - publish-taquito-plugin-to-npm
+      - publish-sdk-to-npm
     uses: ecadlabs/taqueria/.github/workflows/npm-publish.yml@main
     with:
       dir: taqueria-plugin-flextesa
     secrets:
       NPM_TOKEN: ${{ secrets.NPMJS_PAT }}
 
+  run-tests:
+    needs: build-binaries
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./tests
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - name: get binary and make executable from previous step for tests
+        run: |
+          wget ${{ needs.build-binaries.outputs.public-url-ubuntu }} -O taq
+          chmod +x taq
+
+      - name: install dependencies for e2e tests in root and test directories
+        run: |
+          cd ..
+          curl -fsSL https://deno.land/x/install/install.sh | sh
+          npm ci
+          npm install --workspace=taqueria-plugin-ligo
+          npm install --workspace=taqueria-plugin-mock
+          
+          cd tests/
+          npm ci
+      
+      - name: Run unit tests
+        run: |
+          export PATH="/home/runner/.deno/bin:$PATH"
+          npm run test:unit
+
+      - name: Run integration tests
+        run: |
+          export PATH="$(pwd):$PATH"
```

Comment:
Why do you need to update `PATH` here?

Reply:
@danielelisi I did not know there is an env var for path available in these runners. I have updated the starting step that gets the binary to set path for all subsequent steps

Reply:
Sounds good. Please remove all the redundant `export PATH` on lines `174`, `179`, `184`